systemd/rudevil.service

```diff
@@ -10,6 +10,7 @@ ExecStart=/opt/rudevil/rudevil
 Environment=RUDEVIL_USER="root"
 Environment=RUDEVIL_GROUP="plugdev"
 Environment=RUDEVIL_WORKDIR="/storage:
+Environment="DISPLAY=:0"
```

Comment:
Since this is hardcoded, maybe add docs for what it is and how to change it? Considering you're using features for this app, it seems like you want to make it user-facing.

Reply:
display=:0 is a default display for one user environments, it will work only for `feature=notifications`, and for multiuser environments people can change it to whatever, technically, even use it over vnc or something similar